eb

```diff
@@ -39,48 +39,35 @@
 # @author: Pieter De Baets (Ghent University)
 # @author: Jens Timmerman (Ghent University)
 
+PYTHON=python
+
 # Python 2.6 or more recent 2.x required
 REQ_MAJ_PYVER=2
 REQ_MIN_PYVER=6
 REQ_PYVER=${REQ_MAJ_PYVER}.${REQ_MIN_PYVER}
 
 # make sure Python version being used is compatible
-pyver=`python -V 2>&1 | cut -f2 -d' '`
+pyver=`$PYTHON -V 2>&1 | cut -f2 -d' '`
 pyver_maj=`echo $pyver | cut -f1 -d'.'`
-pyver_min=`echo $pyver | cut -f2 -d'.'`
+
+if [ $pyver_maj -ne $REQ_MAJ_PYVER ]
+then
+    PYTHON=${PYTHON}$REQ_MAJ_PYVER}
+    pyver=`$PYTHON -V 2>&1 | cut -f2 -d' '`
+    pyver_maj=`echo $pyver | cut -f1 -d'.'`
+fi
 
 if [ $pyver_maj -ne $REQ_MAJ_PYVER ]
 then
     echo "ERROR: EasyBuild is currently only compatible with Python v${REQ_MAJ_PYVER}.x, found v${pyver}" 1>&2
     exit 1
 fi
+
+pyver_min=`echo $pyver | cut -f2 -d'.'`
 if [ $pyver_min -lt $REQ_MIN_PYVER ]
 then
     echo "ERROR: EasyBuild requires Python v${REQ_PYVER} or a more recent v${REQ_MAJ_PYVER}.x, found v${pyver}." 1>&2
     exit 2
 fi
 
-
-main_script_base_path="easybuild/main.py"
-python_search_path_cmd="python -c \"import sys; print ' '.join(sys.path)\""
-
-easybuild_main=
-for path in `eval $python_search_path_cmd`;
-do
-    test_path="$path/$main_script_base_path"
-    if [ -f $test_path ]
-    then
-        easybuild_main=$test_path
-        break;
-    fi
-done
-
-if [ -z $easybuild_main ]
```

Comment:
all this crap is no longer needed with Py2.6 and newer, since `-m easybuild.main` just works